feature-detects/mediastream.js

```diff
@@ -0,0 +1,13 @@
+define(['Modernizr'], function (Modernizr) {
+    Modernizr.addTest('mediarecorder', function(){
```

Comment:
[addTest](https://modernizr.com/docs#Modernizr.addTest) requires you either give a value, or a function that returns a value. Your function logs out information, but it isn't actually returning anything, and as a a result isn't actually doing anything (from Modernizr's perspective). You would want to either return a value in the function, or, in this case (as it is a simple check) just have the test be inline rather than using a function.